docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -43,97 +37,127 @@ ref over User1, Orderer
+        group enclave endorsement validation transaction
+            CLI -> ECC1 ++: invoke "validateEnclaveEndorsement" ({ ccrsp_msg })
+                group enclave endorsement validation
+                    ECC1 -> ECC1 : <ECC_result, read/writeset, proposal, Sig_Enclave, Enclave_Id> <- extract from  ccrsp_msg
+  
+                    create participant ERCC order 30
+                    ECC1 -> ERCC++ : queryEnclaveCredentials(CC_Id, Enclave_Id)
+                        note right
+                            If FPC enclave is registered, ERCC will
+                            return the corresponding Credentials,
+                            otherwise Credentials is empty and validation will abort.
+                        end note
+                    return Credentials
+                    ECC1 -> ECC1 : extract Enclave_VK from Credentials.attesated_data
+                    ECC1 -> ECC1 : verify Sig_Enclave over <proposal, read/writeset, ECC_result> with Enclave_VK
+                    ECC1 -> ECC1 : extract CC_Params from Credentials.AttestedData
+                    ECC1 -> ECC1 : check CC_Params matches chaincode definition
+                    note right of ECC1
+                        All these check validate for FPC "light" the implicitly defined "designated enclave" 
+                        enclave endorsement policy, the only enclave endorsement policy supported in MVP.
+                    end note
+                end group
+                group transaction replay
+                    ECC1 -> ECC1 : re-create reads and writes based on read/writeset from input proposal
+                end group
+            return validation-proposal-response
+        end group
+        loop for all/"enough" enclave endorsers
+            note right CLI
+                Invoke at enough endorsing peers to satisfy validation endorsement policy
+                and collect { validation-proposal-response }.
+            end note
+            ref over CLI,ECC2,ECC3
+                enclave endorsementt validation transaction
+            end ref
+        end
+    end
 
-    User1 -> User1 : form transaction
+end group
 
-    User1 -> Orderer++ : send transaction
-    deactivate User1
+group transaction submission
+    CLI -> CLI : form transaction from { validation-proposal-response } (FPC "light") or from { proposal-response } (otherwise)
 
+    CLI -> Orderer++ : send transaction
+    deactivate CLI
+    ...
+    Orderer -> Orderer : form block
 end group
 
-    group collect transactions
-        ...
-        Orderer -> Orderer : form block
-    end group
-
-    Orderer -> ECC1+ : broadcast
-    & Orderer -> ECC2+ : broadcast
-    & Orderer -> ECC3+ : broadcast
-    deactivate Orderer
+Orderer -> ECC1+ : broadcast
+& Orderer -> ECC2+ : broadcast
+& Orderer -> ECC3+ : broadcast
+deactivate Orderer
 
-    group validation & commit
-        ECC1 -> ECC1 : validate block signature
-        loop for each tx in block
+group validation & commit
```

Comment:
This entire block is "Post-MVP", as we have no FPC-logic built into the peer.
So, there is nothing to specify here.
```suggestion
group Post-MVP: validation & commit
```

Reply:
not really true: even for FPC Lite the validation and commit has to happen, just only in the `default validation` group, not in `fpc-vscc`, and the final `apply valid tx writeset` (but not in TL)

Reply:
> not really true: even for FPC Lite the validation and commit has to happen

of course it has to, but that's vanilla Fabric, so there is nothing here that add or remove.
Anyway, minor.